### PR TITLE
Fix: Remove "not RFC1918 IPs" from examples.

### LINKED ISSUE #3169: Remove not RFC1918 IPs from examples
This IP scope is not RFC1918.

              This scenario is a verbatim copy of our example `dual-dc-l3ls`
https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/FABRIC.yml#L75
do you want me to fix it there as well?

_Originally posted by @gmuloc in https://github.com/aristanetworks/ansible-avd/pull/2907#discussion_r1337086411_

### PR DESCRIPTION
**Pull request by Shivani-gslab**
## Change Summary

Remove not RFC1918 IPs from examples.

## Related Issue(s)

Fixes #3169 

## Component(s) name

`arista.avd.examples`

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/FABRIC.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ l3_edge:
   # Define a new IP pool that will be used to assign IP addresses to L3 Edge interfaces.
   p2p_links_ip_pools:
     - name: DCI_IP_pool
-      ipv4_pool: 172.100.100.0/24
+      ipv4_pool: 172.16.100.0/24
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/FABRIC.yml:75]: you will need to fix the documentation as well to be aligned:
https://avd.sh/en/stable/examples/dual-dc-l3ls/index.html#setting-fabric-wide-configuration-parameters

File: ansible_collections/arista/avd/examples/campus-fabric/group_vars/DC1.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ aaa_authorization:
     default: local
 
 # OOB Management network default gateway.
-mgmt_gateway: 172.100.100.1
+mgmt_gateway: 172.16.100.1
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/campus-fabric/group_vars/DC1.yml:17]: need to fix documentation here: https://avd.sh/en/stable/examples/campus-fabric/index.html

File: ansible_collections/arista/avd/examples/isis-ldp-ipvpn/group_vars/WAN1.yml [hunk 3]
```diff
@@ -101,7 +101,7 @@ core_interfaces:
   # First, an IP-pool for the underlay p2p links is defined
   p2p_links_ip_pools:
     - name: core_pool
-      ipv4_pool: 100.64.48.0/24
+      ipv4_pool: 10.255.3.0/24
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/isis-ldp-ipvpn/group_vars/WAN1.yml:104]: Need to fix documentaiton here: https://avd.sh/en/stable/examples/isis-ldp-ipvpn/index.html#ip-ranges-used (2 occurences of 100.64)

File: ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1.yml [hunk 4]
```diff
@@ -21,7 +21,7 @@ local_users:
     sha512_password: "$6$eucN5ngreuExDgwS$xnD7T8jO..GBDX0DUlp.hn.W7yW94xTjSanqgaQGBzPIhDAsyAl9N4oScHvOMvf07uVBFI4mKMxwdVEUVKgY/."
 
 # OOB Management network default gateway
-mgmt_gateway: 172.100.100.1
+mgmt_gateway: 172.16.100.1
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1.yml:24]: Need to fix documentation and looking at the doc, you probably need to fix the inventory.yml as well: https://avd.sh/en/stable/examples/l2ls-fabric/index.html#inventoryyml

**Review by gmuloc — CHANGES_REQUESTED** [review]
Thanks for this PR! Put some comments

for each examples the documentation needs to be updated with the new subnets
please also check the inventory.yml file for each to make sure no wrong management IP is left

File: ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/FABRIC.yml (hunk 1 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/FABRIC.yml:75]: Done

File: ansible_collections/arista/avd/examples/campus-fabric/group_vars/DC1.yml (hunk 2 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/examples/campus-fabric/group_vars/DC1.yml:17]: Done

File: ansible_collections/arista/avd/examples/isis-ldp-ipvpn/group_vars/WAN1.yml (hunk 3 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/examples/isis-ldp-ipvpn/group_vars/WAN1.yml:104]: Done

File: ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1.yml (hunk 4 above)
Comment (Shivani-gslab) [reply ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1.yml:24]: Done

**Review by gmuloc — APPROVED** [review]
LGTM - thanks for this and good catch for the basic-configurations ;)

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**